testing_and_setup/compass/ocean/drying_slope/marsh_flooding/idealized_transect/config_forward1.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<config case="forward1">
+	<add_link source="../init_step1/ocean.nc" dest="init.nc"/>
+	<add_link source="../init_step1/forcing.nc" dest="forcing.nc"/>
+	<add_link source="../init_step1/graph.info" dest="graph.info"/>
+
+	<namelist name="namelist.ocean" mode="forward">
+		<template file="forward_template.xml" path_base="script_configuration_dir"/>
+		<template file="forward_idealized_transect.xml" path_base="script_configuration_dir"/>
+	</namelist>
+
+	<streams name="streams.ocean" keep="immutable" mode="forward">
+		<template file="forward_template.xml" path_base="script_configuration_dir"/>
+	</streams>
+
+	<run_script name="run.py">
+		<step executable="gpmetis">
+			<argument flag="graph.info">32</argument>
+		</step>
+		<model_run procs="32" threads="1" namelist="namelist.ocean" streams="streams.ocean"/>
```

Comment:
This could be changed to 36 so that it uses a full node on Grizzly.

Reply:
fixed.